lib/runtime/wazero/instance.go

```diff
@@ -107,7 +107,7 @@ func NewInstance(code []byte, cfg Config) (instance *Instance, err error) {
 
 	_, err = rt.NewHostModuleBuilder("env").
 		// values from newer kusama/polkadot runtimes
-		ExportMemory("memory", 23).
+		ExportMemory("memory", allocator.MaxWasmPages-1).
```

Comment:
Why the `-1` ?

Reply:
The substrate wasm runtime supports u64 but wazero don't. So the max wasm pages value is `65536` and each page has `65536` bytes if you multiply the total bytes will be `4294967296` which overflows the uint32 by one, so I setup the `minPages` to be `65535`

Reply:
Great! thanks for the explanation!